components/clarity-lsp/src/common/requests/completion.rs

```diff
@@ -80,20 +97,27 @@ impl<'a> ContractDefinedData {
         &mut self,
         expr: &SymbolicExpression,
         name: &ClarityName,
-        parameters: &Vec<TypedVar<'a>>,
+        parameters: &Option<Vec<TypedVar<'a>>>,
```

Comment:
Yes i initially tried to favorise empty vecs.

This method kind of does two things (the second being populating function_completion_items), and because of the `if let Some(parameters)` I had before, it wasn't called for functions with no arguments. So this was the simple way to handle it.
I might need to split it into two functions. I'll give it a try before merging